lisa/features/disks.py

```diff
@@ -68,6 +69,9 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
     def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
+    def get_linux_device_luns(self) -> Dict[str, int]:
```

Comment:
`get_luns`

Reply:
The method gets the luns from within the VM. So I felt its better if we can put that in the same. 
get_linux_luns seems more dev friendly to me.

Reply:
I know the logic applies to `linux` only, but we can extend it if needed. So the name could be called `get_luns`.

Reply:
This method specifically brought to cross check if luns from linux are matching with azure or not. If there is any such method needed for azure we can name it accordingly.

Reply:
This is in base class, maybe extended only in downstream. So make all method names to be right at first time.

Reply:
Changed the method name to get_luns